pages.de/common/jq.md

```diff
@@ -0,0 +1,32 @@
+# jq
+
+> Ein JSON-Verarbeiter für die Kommandozeile mit einer domänenspezifischen Sprache.
+> Weitere Informationen: <https://stedolan.github.io/jq/manual/>.
+
+- Führe den angegebenen Ausdruck aus (gib farbiges und formatiertes JSON aus):
+
+`{{cat pfad/zu/datei.json}} | jq '{{.}}'`
```

Comment:
The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/jq.md) page should be change before this fix is applied here:
```suggestion
`{{cat pfad/zu/datei.json}} | jq '.'`
```

Optional change: our style guide doesn't handle such cases.

Reply:
If you think that this should be changed, please open a separate PR changing it in all languages.